ow_lander/action/Stop.action

```diff
@@ -0,0 +1,8 @@
+#goal definition
+std_msgs/Empty goal
+---
+# result
+geometry_msgs/Point final     # arm final position
+---
+# feedback
+geometry_msgs/Point current   # arm current position
```

Comment:
OWLAT's ARM_STOP has no result or feedback.  We should follow suit, or argue that these are useful.  If the arm's current position is available in telemetry, it may not be so useful here.